src/Renci.SshNet/Sftp/SftpFileAttributes.cs

```diff
@@ -544,10 +582,12 @@ internal static SftpFileAttributes FromBytes(SshDataStream stream)
 
             if ((flag & 0x00000008) == 0x00000008)   //  SSH_FILEXFER_ATTR_ACMODTIME
             {
+                // The incoming times are "Unix times", so they're already in UTC.  We need to preserve that
+                // to avoid losing information in a local time conversion during the "fall back" hour in DST.
                 var time = stream.ReadUInt32();
-                accessTime = DateTime.FromFileTime((time + 11644473600) * 10000000);
+                accessTime = DateTime.FromFileTimeUtc((time + 11644473600) * 10000000);
                 time = stream.ReadUInt32();
-                modifyTime = DateTime.FromFileTime((time + 11644473600) * 10000000);
+                modifyTime = DateTime.FromFileTimeUtc((time + 11644473600) * 10000000);
```

Comment:
Please remove the default value of **DateTime.MinValue**, and add an else block in which **accessTime** is initialized to `DateTime.SpecifyKind(DateTime.MinValue, DateTimeKind.Local)`.

Reply:
Done, but I specified DateTimeKind.Utc from the else block.